mathesar/api/exceptions/exception_mappers.py

```diff
@@ -0,0 +1,16 @@
+from psycopg2.errors import CheckViolation, UniqueViolation
+
+from mathesar.api.exceptions.database_exceptions import (
+    exceptions as database_api_exceptions,
```

Comment:
why do you need to rename these?

Reply:
We might be importing from various exception modules which are similar in name